README.md

```diff
@@ -144,25 +161,30 @@ Ensure that Python is installed on your machine before proceeding.
 
 8. Create a `active_usernames.txt` file alongside the `active_usernames.example.txt` file and add all the usernames that
    can use this system. The usernames are the Federal IDs and each one should be stored on a separate line.
+
    ```bash
    cp active_usernames.example.txt active_usernames.txt
    ```
 
 9. Start the microservice using Uvicorn:
+
    ```bash
    uvicorn ldap_jwt_auth.main:app --log-config ldap_jwt_auth/logging.ini --reload
    ```
+
    The microservice should now be running locally at http://localhost:8000. The Swagger UI could be accessed
    at http://localhost:8000/docs.
 
 10. To run the unit tests, run:
-   ```bash
-   pytest -c test/pytest.ini test/unit/
-   ```
+
+```bash
+pytest -c test/pytest.ini test/unit/
+```
```

Comment:
What happened to the formatting here?

Reply:
vs code seems to autoformat it whenever I hit save

Reply:
Yeah that is weird, mine also does that. Also interesting to note the syntax highlighting for the commands only works when the spaces are removed. Its fine as it is.